src/browser/atom-window.coffee

```diff
@@ -49,6 +49,8 @@ class AtomWindow
     loadSettings.devMode ?= false
     loadSettings.safeMode ?= false
 
+    @devMode ?= false
```

Comment:
without this, the check in `AtomApplication::windowForPaths` was failing: it tried to compare `null` with `undefined` and coffeescript doesn't see them as close enough to pass the `is` test.